src/Symfony/Component/Process/Process.php

```diff
@@ -596,8 +603,6 @@ public function clearErrorOutput()
     public function getExitCode()
     {
         if (!$this->enhanceSigchildCompatibility && $this->isSigchildEnabled()) {
-            $this->stop(0);
```

Comment:
these have been added on 2.3 but not yet tagged in any version. To do backported to 2.3 asap